ikasaneip/security/db/src/main/java/org/ikasan/security/model/RoleModule.java

```diff
@@ -0,0 +1,67 @@
+package org.ikasan.security.model;
+
+import java.util.Date;
+
+public class RoleModule
```

Comment:
Given you storing RoleModule objects in a set you need a equals and hashCode implementation.

Reply:
done